bears/vcs/git/GitCommitBear.py

```diff
@@ -2,11 +2,12 @@
 import shutil
 
 from bears.vcs.CommitBear import _CommitBear
+from bears.vcs.VCSCommitBear import VCSCommitBear
 from coala_utils.ContextManagers import change_directory
 from coalib.misc.Shell import run_shell_command
 
 
-class GitCommitBear(_CommitBear):
+class GitCommitBear(_CommitBear, VCSCommitBear):
```

Comment:
You need to add 
```
        try:
            super(_CommitBear,self).run()
        except NotImplementedError as e:
            pass
```
within the run method of _CommitBear so that it can call VCSCommitBear's run method as well